src/zenml/integrations/aws/orchestrators/sagemaker_orchestrator.py

```diff
@@ -859,12 +859,16 @@ def fetch_status(self, run: "PipelineRunResponse") -> ExecutionStatus:
         )["PipelineExecutionStatus"]
 
         # Map the potential outputs to ZenML ExecutionStatus. Potential values:
-        # https://cloud.google.com/vertex-ai/docs/reference/rest/v1beta1/PipelineState
-        if status in ["Executing", "Stopping"]:
+        # https://docs.aws.amazon.com/sagemaker/latest/APIReference/API_DescribePipelineExecution.html
+        if status == "Executing":
             return ExecutionStatus.RUNNING
-        elif status in ["Stopped", "Failed"]:
+        elif status == "Stopping":
+            return ExecutionStatus.STOPPING
+        elif status == "Stopped":
+            return ExecutionStatus.STOPPED
```

Comment:
Just checking: there is no implementation for `_stop` in this orchestrator, so these statuses would only be set if the user manually stopped the Sagemaker pipeline using the AWS console or CLI, correct ?